connect/proxy/config.go

```diff
@@ -85,43 +84,21 @@ func (plc *PublicListenerConfig) applyDefaults() {
 	}
 }
 
-// UpstreamConfig configures an upstream (outgoing) listener.
-type UpstreamConfig struct {
-	// LocalAddress is the host/ip to listen on for local app connections. Defaults to 127.0.0.1.
-	LocalBindAddress string `json:"local_bind_address" hcl:"local_bind_address,attr" mapstructure:"local_bind_address"`
+// UpstreamConfig is an alias for api.Upstream so we can parse in a compatible
```

Comment:
Whoa. I didn't know type aliasing let you add your own methods on the aliased type.